week3/body_mass.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
+
+'''
+Body Mass Index (BMI) is a measure of health on weight.
+It can be calculated by taking your weight in kilograms
+and dividing by the square of your height in meters.
+
+>>> bmi_calculator(65, 1.65)
+'Normal'
+'''
+
+def bmi_calculator(weight, height):
+
+    BMI = weight / height**2
+
+    TABLE = (
+        ((29.9), "Obese"),
```

Comment:
you do have to put 29.9 in parentheses, it introduces more complexity I believe (less readability), which we do not want.